src/main/java/com/openshift/jenkins/plugins/pipeline/OpenShiftBuildVerifier.java

```diff
@@ -46,18 +41,6 @@ public String getCheckForTriggeredDeployments() {
 		return checkForTriggeredDeployments;
 	}
 	
```

Comment:
another rebase hit with @oatmealraisin - he though has already moved get to a default method within a new interface with his PR ... maybe you guys can link up and coordinate this aspect of the change to ease the rebase pain